grudge/geometry/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+from grudge.geometry.metrics import *
```

Comment:
Not specific to this, but a general comment: For review, I'm kind of wary of letting this thing grow too big. I'd be more in favor of merging it incrementally.

Reply:
> Would the next logical thing be to split out the geometry computations?

Yeah, I think that would make sense.